eventmesh-sdk-java/src/main/java/org/apache/eventmesh/client/grpc/consumer/SubStreamHandler.java

```diff
@@ -159,19 +159,15 @@ public void close() {
 
     private void senderOnNext(final Subscription subscription) {
```

Comment:
My suggestion is to put 'synchronized' on the method. Do you have any better suggestions?

Reply:
Yes I am thinking same of putting back the 'synchronized' on the method